Library/Formula/gdk-pixbuf.rb

```diff
@@ -21,6 +21,9 @@ class GdkPixbuf < Formula
   depends_on "libpng"
   depends_on "gobject-introspection"
 
+  # disable the default relocatable library setting
+  patch :DATA
```

Comment:
Would be perhaps nice to make this an `inreplace` if we're going to have to carry it indefinitely.